modules/plotting/include/modules/plotting/datastructures/dataframeutil.h

```diff
@@ -52,6 +52,9 @@ std::shared_ptr<plot::DataFrame> IVW_MODULE_PLOTTING_API
 combineDataFrames(std::vector<std::shared_ptr<plot::DataFrame>> histogramTimeDataFrame,
                   bool skipIndexColumn = false, std::string skipcol = "index");
 
+std::string IVW_MODULE_PLOTTING_API createToolTipForRow(const plot::DataFrame &dataframe,
```

Comment:
Would be nice if one could specify which columns to show

Reply:
I agree, and that is something I probably will implement later when I need it, in another PR. I added the util-function so that it would be easier to do. 
Will also add tooltip to the scatterplot in the future.